mobly/records.py

```diff
@@ -607,11 +608,31 @@ def is_test_executed(self, test_name):
         return True
     return False
 
+  def _calculate_retry_count_delta(self):
```

Comment:
something along the lines of "eventually_passing_retries" might be a more intuitive name